### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jmprieur**
- Improving the readme (structure, pictures, +explanations, spelling, grammar, style)
- changing the sample to use Microsoft Graph rather than AAD Graph

### CODE REVIEW

File: AppCreationScripts/AppCreationScripts.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Registering the Azure Active Directory applications and updating the configuration files for this sample using PowerShell scripts
+
+## Goal of the scripts
```
Comment (henrik-me) [inline AppCreationScripts/AppCreationScripts.md]: Does this script come from the same source as all the other samples?

File: DirectorySearcherLib/DirectorySearcher.cs [hunk 2]
```diff
@@ -48,7 +48,7 @@ public static class DirectorySearcher
 
             try
             {
-                string graphRequest = String.Format(CultureInfo.InvariantCulture, "{0}/{1}/users?api-version={2}&$filter=mailNickname eq '{3}'", graphResourceUri, authResult.TenantId, graphApiVersion, alias);
+                string graphRequest = String.Format(CultureInfo.InvariantCulture, "{0}/v1.0/users?$filter=mailNickname eq '{3}'", graphResourceUri, authResult.TenantId, graphApiVersion, alias);
```
Comment (henrik-me) [inline DirectorySearcherLib/DirectorySearcher.cs]: >0}/v1.0/users?$filter=mailNickname eq '{3}' [](start = 84, length = 43)

you are only using param 0 and 3. consider rewriting this and remove the no longer needed info. Also you can use the $"" format to make it more readable?

File: README.md [hunk 3]
```diff
@@ -1,80 +1,134 @@
...
+### Scenario
 
-The `DirectorySearcherLib` PCL is the application's shared C# code base, which contains both the identity related logic and the search related logic.  It leverages ADAL.NET v3, which is also a PCL, to automatically sign users in with the OAuth 2.0 protocol, acquire an access token for the AAD Graph API, cache tokens, maintain user sessions, and so on.  It then queries AAD's Graph API for information about a user in the authenticated user's tenant with a matching alias, and returns results to the platform specific UI code.  By writing both the identity and search logic in the `DirectorySearcherLib` PCL, the code only needs to be written once and can be reused across each platform.
+The user enters an alias in the organization of interest an presses search. If needed, the user is asked to sign in to that organization and to consent for the application to read user's basic profile.
```
Comment (henrik-me) [inline README.md]: >an [](start = 57, length = 2)

and ?

File: README.md [hunk 4]
```diff
@@ -1,80 +1,134 @@
...
 
-To run this entire sample you will need:
 - An Internet connection
 - A Windows machine (necessary if you want to run the Windows Store or WPF apps)
 - An OS X machine (necessary if you want to run the iOS app)
 - Visual Studio 2017 (recommended) which comes with Xamarin for iOS, Android, and Universal platform;
 - An Azure subscription (a free trial is sufficient)
 
 Every Azure subscription has an associated Azure Active Directory tenant.  If you don't already have an Azure subscription, you can get a free subscription by signing up at [https://azure.microsoft.com](https://azure.microsoft.com).  All of the Azure AD features used by this sample are available free of charge.
+This sample will not work with a Microsoft account (formerly Windows Live account). Therefore, if you signed in to the [Azure portal](https://portal.azure.com) with a Microsoft account and have never created a user account in your directory before, you need to do that now.
```
Comment (henrik-me) [inline README.md]: >Microsoft account [](start = 167, length = 17)

does this really have to be an MSA account? I could sign-in using my work account to create a dictionary?

Comment (henrik-me) [inline README.md]: >Microsoft account [](start = 167, length = 17)

Minor:
Some places it says Microsoft account. Other places it says Microsoft Account.

Comment (jmprieur) [reply README.md]: this won't work with an MSA.

File: AppCreationScripts/AppCreationScripts.md (hunk 1 above)
Comment (jmprieur) [reply AppCreationScripts/AppCreationScripts.md]: yes. Why?

**Pull request merged**